features/buyer/requirements.feature

```diff
@@ -196,3 +196,33 @@ Scenario: There is no 'Publish requirements' button for an incomplete requiremen
   And I have created an individual specialist requirement
   When I click 'Review and publish your requirements'
   Then I don't see the 'Publish requirements' button
+
+
+Scenario: Ensure we can log in as a buyer with a closed brief
```

Comment:
Oh wow, maybe it only works at the beginning of the file?
I never thought about it before, but kinda makes sense now @_@